config/routes.rb

```diff
@@ -3,4 +3,13 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  # get "/", to: "ingredient#welcome"
+
+  get "/ingredients", to: "ingredients#index"
+
+  get "/recipes/:id", to: "recipes#show"
+  post "/recipes/:id", to: "recipes#create"
```

Comment:
This POST path is for creating an association between recipes and ingredients, so a more RESTful and representative path would include the names of both of these resources. Consider a path like `"/recipe_ingredients"` or `"/recipes/:id/ingredients"`

And then, based on the earlier comment around controller organization, this could direct to a joins controller. Taken together, the more RESTful option for this route would be 
`post "/recipes/:id/ingredients", to: "recipe_ingredients#create"`